README.md

```diff
@@ -9,6 +9,8 @@ A release is a version of your code that is deployed to an environment. When you
 
 Additionally, releases are used for applying source maps to minified JavaScript to view original, untransformed source code. You can learn more about releases in the [releases documentation](https://docs.sentry.io/workflow/releases).
 
+As an alternative to using this action, you can install the [GitHub Integration](https://sentry.io/integrations/github/) to tell Sentry about your releases.
```

Comment:
@mgaeta We should also mention that you need to use the CLI to send releases in conjunction with the integration (https://docs.sentry.io/cli/releases/). It's not enough to just have the integration installed.